azurerm/internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -49,6 +49,18 @@ func resourceArmLogAnalyticsWorkspace() *schema.Resource {
 
 			"resource_group_name": azure.SchemaResourceGroupNameDiffSuppress(),
 
+			"public_network_access_for_ingestion_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+
+			"public_network_access_for_query_enabled": {
```

Comment:
"public network" is a strange phrase here, if this is referring to enabling queries over the public internet - could this have a more descriptive name?

Reply:
it's used to enable queries through public DNS. I will change the name to `enable_query_over_public_dns`. is it fine?